configure.sh

```diff
@@ -16,10 +16,12 @@ Configures the CMAKE build environment.
 --yices2                build yices2            (default: off)
 --cvc4                  build cvc4              (default: off)
 --msat                  build MathSAT           (default: off)
+--z3					build z3				(default: off)
```

Comment:
minor: line up "build" with other rows

Reply:
So strangely enough this is lined up on my local master but isn't registering as a change that I should add/commit. Thoughts on how to fix that?

Reply:
Oh then it's probably a tabs vs spaces issue. If you show whitespace characters, there are probably tabs there which you can replace with spaces to make sure it's lined up on other editors. https://stackoverflow.com/questions/6255839/show-space-tab-crlf-characters-in-editor-of-visual-studio

Reply:
You can also tell your editor to always do spaces when you hit tab. 
https://stackoverflow.com/questions/29972396/how-can-i-customize-the-tab-to-space-conversion-factor

And there is an extension to replace existing tabs with spaces.
https://marketplace.visualstudio.com/items?itemName=johnnywong.vscode-ts-whitespace-converter

Reply:
thanks! (They were tabs, whereas all the others were spaces!)